templates/credentials/base_cred.html

```diff
@@ -9,6 +9,58 @@
 {% endblock body_class %}
 
 {% block outer_content %}
+  {% if show_cred_maintenance_alert and cred_maintenance_start and cred_maintenance_end %}
+    <div class="p-notification--information" style="margin:1rem">
+      <div class="p-notification__content">
+        <h5 class="p-notification__title">Maintenance Scheduled</h5>
+        <p class="p-notification__message" id="cred-maintenance-message"></p>
+      </div>
+    </div>
+    <script>
+      const formatter = (dateTime, locale = 'default') => {
+        const date = new Date(dateTime);
+        const options = {
+          year: 'numeric',
+          month: '2-digit',
+          day: '2-digit',
+          hour: '2-digit',
+          minute: '2-digit',
+          hour12: true,
+          timeZoneName: 'short',
+        };
+
+        let formattedDate = date.toLocaleString(locale, {
+          ...options,
+          timeZoneName: undefined
+        });
+        formattedDate = formattedDate.replace(/am|pm/, match => match.toUpperCase());
+        const timeZone = date.toLocaleString(locale, {
+          timeZoneName: 'short'
+        }).split(' ').pop();
+        return `${formattedDate} (${timeZone})`;
+      }
+
+      const handleFormatMaintenanceMessage = () => {
+        const showBanner = '{{ show_cred_maintenance_alert }}';
+        const isUnderMaintenance = '{{ cred_is_in_maintenance }}';
+        if (showBanner === 'True') {
```

Comment:
This can be simply this to avoid nested if 
`if (showBanner !== 'True') return;`